app/models/property.rb

```diff
@@ -0,0 +1,12 @@
+require 'data_mapper'
+require 'dm-postgres-adapter'
+
+class Property
+
+include DataMapper::Resource
+
+property :id,       Serial
+property :name,     String
```

Comment:
Please change prop_name, to avoid confusion with username (we will also change).